README.md

```diff
@@ -24,16 +24,21 @@ Select the test versions to run PHPCompatibility. It supports any format accepte
 
 Paths to run compatibility check on. Default: `${{ github.workspace }}`
 
+### extensions
+
+Specify the extensions of the files that will be checked. Example: `extensions: php,inc,lib`. By default, it uses the options set by PHPCompatibility and PHP_CodeSniffer.
+
 ## Example Usage
 
 ```
 uses: pantheon-systems/phpcompatibility-action@v1
 with:
   test-versions: 7.4-
   paths: ${{ github.workspace }}/src
+  extensions: php
```

Comment:
if we're defaulting to `php`, then we either don't need to show the extensions in the example at all, or else show the extensions input with different filetypes.

```suggestion
  extensions: 'php,inc'
```